app/views/card_transactions/_category_transaction_fields.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="nested-form-wrapper" data-new-record="<%= form.object.new_record? %>">
+  <% if form.object&.category&.built_in? %>
```

Comment:
The lack of this if else would cause an Exchange category, for example, to be rewritten as something else in a update, because @entities does not include built_ins.